include/time.h

```diff
@@ -25,15 +25,19 @@
 #ifndef KTF_TIME_H
 #define KTF_TIME_H
 
+#define MS(s) (s * 1000)
+#define uS(s) (MS(s) * 1000)
+#define NS(s) (uS(s) * 1000)
+
 typedef uint64_t time_t;
 
-extern void sleep(time_t ms);
+extern void msleep(time_t ms);
 extern time_t get_timer_ticks(void);
 
 /* Static declarations */
 
-static inline void msleep(time_t ms) {
-    sleep(ms);
+static inline void sleep(time_t s) {
+    msleep(MS(s));
```

Comment:
I find `msleep(s * 1000)` easier to read (note that this doesn't need the `ULL` suffix, as `s` is already correctly typed). But maybe it's just me.

Reply:
Ok